xapian-applications/omega/query.cc

```diff
@@ -1420,9 +1423,14 @@ eval(const string &fmt, const vector<string> &param)
 		if (!value.empty()) value.erase(value.size() - 1);
 		break;
 	    }
-	    case CMD_chr:
-		Xapian::Unicode::append_utf8(value, string_to_int(args[0]));
+	    case CMD_chr: {
+		unsigned int temp;
+		if (!parse_unsigned(args[0].c_str(), temp)) {
+		    throw "Unicode codepoint for command chr should be >= 0";
+		}
+		Xapian::Unicode::append_utf8(value, temp);
```

Comment:
You really do like your `temp` variable names, don't you?

Here e.g. `codepoint` would be a better name than `temp`.